syweb/webclient/lib/matrix.js

```diff
@@ -296,6 +297,24 @@ var init = function(exports){
             );
         },
 
+        getThreePids: function(creds, bind, callback) {
+            var path = "/account/3pid";
+            return this._doAuthedV2Request(
+                callback, "GET", path, undefined, undefined
+            );
+        },
```

Comment:
I have added them: see PR in matrix-js-sdk. I re-copied the file from the changed matrix-js-sdk.